components/branding/LogoBlock.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { Card, CardContent } from '@mui/material/'
+import Image from 'next/image'
+
+// to do: 
+// consider using usePathname and our custom 
+// Link component when upgrading to Nextjs v13
+
+const styles = {
+  card: {
+    padding: 0,
+    margin: '16px'
+  },
+  topper: {
+    borderBottom: `1px solid grey`,
+  },
+  content: {
+    backgroundColor: 'white',
+  },
+}
+
+export const LogoBlock = ({logo, bgColor}) => {
+  return (
+    <Card sx={ styles.card }>
+      <CardContent sx={ styles.topper } style={{ backgroundColor: bgColor }}>
```

Comment:
we should stick with the `sx` prop for these MUI components and not mix `sx` and `style`